article/app/views/fragments/articleBody.scala.html

```diff
@@ -36,24 +36,47 @@
             }
 
             @if(amp) {
+
+                @fragments.mainMedia(article, amp)
+
                 @fragments.headTonal(article, model, isPaidContent, amp = amp)
 
-                @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                    @fragments.mainMedia(article, amp)
-                }
             } else {
-                <div class="hide-on-mobile">
-                    @fragments.headTonal(article, model, isPaidContent, amp = amp)
+                @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
 
-                    @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                        @fragments.mainMedia(article, amp)
-                    }
-                </div>
-                <div class="mobile-only">
-                    @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                        @fragments.mainMedia(article, amp)
-                    }
-                </div>
+                    <div class="content__header-container">
+                        @fragments.headTonal(article, model, isPaidContent)
+
+                        <div class="content__main-media-full-width-mobile">
+                            @fragments.mainMedia(article)
+                        </div>
+                    </div>
+
+                } else {
+                    <div class="content__header-container">
+                        @fragments.headTonal(article, model, isPaidContent)
+
+                        <div class="content__main-media-wrapper content__main-media-full-width-mobile">
+                            <div class="content__main-media">
+
+                                @fragments.mainMedia(article)
+                            </div>
+
+                            <div class="hide-on-mobile">
```

Comment:
Tried that. They have to be in the same container to use flex & order. That is particularly difficult as they have to appear below the sport/football scores on mobile, but contained within the same container for main media on desktop. It would be more work and not entirely sure its possible. I can test out in a separate PR - but would rather have an idea of how the desktop article layout may change first.